src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,85 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
```

Comment:
I am not sure we need unit tests for this kind of class. AmazonAthenaDataSourceItem does not change any fields / behaviour of its parent class. 
How do you think @hainv-ohio ?

Reply:
It's true, as you said, but maybe only for now, in the future, that may change, so I think it's fine to keep the tests here. But @recca5p I don't think we need more tests than testing the happy case for the constructor. If we need to test the edge cases, we can test them in the parent class's tests.